File: 3-operators/operators.rb
#!/bin/ruby

require 'json'
require 'stringio'

# Complete the solve function below.
def solve(meal_cost, tip_percent, tax_percent)
    tip = meal_cost*tip_percent/100;
    tax = meal_cost*tax_percent/100;
    totalCost = meal_cost + tip + tax;

    puts totalCost.round
end

meal_cost = gets.to_f

tip_percent = gets.to_i

tax_percent = gets.to_i

solve meal_cost, tip_percent, tax_percent

File: 3-operators/operators.py
#!/bin/python3

def solve(meal_cost, tip_percent, tax_percent):
    tip = float(meal_cost*tip_percent/100)
    tax = float(meal_cost*tax_percent/100)
    totalCost = float(meal_cost + tip + tax)
    print(int(round(totalCost)))


if __name__ == '__main__':
    meal_cost = float(input())

    tip_percent = int(input())

    tax_percent = int(input())

    solve(meal_cost, tip_percent, tax_percent)
